Console/retro/Console.cc

```diff
@@ -352,7 +351,6 @@ void Console::PutCharNoUpdate(char c)
         cursorX++;
         if(cursorX >= cols)
             PutCharNoUpdate('\n');
-        // This is to make sure the cursor width is calculated correctly
         chars[cursorY * cols + cursorX].attrs = currentAttr;
```

Comment:
Why do the attributes for the *next* character need to be changed?

Reply:
I erased the comment, but the following line was not useful, too.